challenges/classes.js

```diff
@@ -1,9 +1,36 @@
 // Copy and paste your prototype in here and refactor into class syntax.
 
+class CuboidMaker {
+  constructor(properties) {
+    this.length = properties.length;
+    this.width = properties.width;
+    this.height = properties.height;
+  }
+  volume (){
+    return (this.length * this.width * this.height);
+  }
+  surfaceArea (){
+    return (2*(length * (width + length) * (height + width) * height));
+  }
+}
+
+const cuboid1 = new CuboidMaker({
```

Comment:
You'll either have to change cuboid1 to cuboid to make the console logs work, or change the variable name in the console logs to be cuboid1 to match this declaration.